extra.js

```diff
@@ -12,8 +12,22 @@
 function calculator(operator, a, b) {
   switch (
     operator
-    // CODE HERE
   ) {
+    case "+" :
+      console.log(a+b)
+      break
+
+    case "-" :
+      console.log(a-b)
+      break
+
+    case "*" :
+      console.log(a*b)
+      break
+
+    case "/" :
+      console.log(a/b)
+      break
   }
 }
 

```

Comment:
use "if statement" to run the switch case only when the inputs(a & b) are positive